src/plugins/Scene3D.cc

```diff
@@ -1105,8 +1111,14 @@ void TextureNode::PrepareNode()
     delete this->texture;
     // note: include QQuickWindow::TextureHasAlphaChannel if the rendered
     // content has alpha.
+#if QT_VERSION < QT_VERSION_CHECK(5, 14, 0)
     this->texture = this->window->createTextureFromId(
         newId, sz, QQuickWindow::TextureIsOpaque);
+#else
+    // TODO(anyone) Use createTextureFromNativeObject
+    this->texture = this->window->createTextureFromId(
+        newId, sz, QQuickWindow::TextureIsOpaque);
```

Comment:
@iche033, homebrew is already using Qt 5.14 and there's a new deprecation warning:

`'createTextureFromId' is deprecated: Use createTextureFromNativeObject() instead`

See [Obsolete Members for QQuickWindow](https://doc.qt.io/qt-5/qquickwindow-obsolete.html) and [QQuickWindow::createTextureFromNativeObject](https://doc.qt.io/qt-5/qquickwindow.html#createTextureFromNativeObject). Do you think that fixing this warning will be quick or does it need more thought? I didn't dig too deep to find out how to get the `nativeObjectPtr`.